sky/backends/cloud_vm_ray_backend.py

```diff
@@ -276,7 +282,16 @@ def check_ip():
                     for i in range(pg.bundle_count)
                 ])
                 print('INFO: Reserved IPs:', ip_list)
-                ip_list_str = '\\n'.join(ip_list)
+
+                if {cluster_ips_sorted!r}:
```

Comment:
nit: always test `is/is not None`: https://google.github.io/styleguide/pyguide.html#2144-decision